lib/streamlit/elements/color_picker.py

```diff
@@ -0,0 +1,74 @@
+import re
+
+from streamlit.proto.ColorPicker_pb2 import ColorPicker as ColorPickerProto
+from streamlit.errors import StreamlitAPIException
+from .utils import _get_widget_ui_value
+
+
+class ColorPickerMixin:
+    def color_picker(dg, label, value=None, key=None):
```

Comment:
~Hm yeah this could honestly go either way. My initial thought was that the final namespacing should occur in the __init__.py file and internally we should keep clear names, but now that you mention it, the docstring right above it calls it out as beta_color_picker so it might be easier to keep consistent that way.~

~I made the other beta API we have inconsistent, so I'll fix both of them to be beta_ in the mixin directly.~

Actually, this is important to keep beta_color_picker since sidebar/empty elements will use the mixin name directly. Thanks for catching.